spanner/api/Spanner.Samples.Tests/ReadStaleDataAsyncTest.cs

```diff
@@ -30,6 +31,7 @@ public async Task TestReadStaleDataAsync()
     {
         ReadStaleDataAsyncSample sample = new ReadStaleDataAsyncSample();
         await _spannerFixture.RefillMarketingBudgetsAsync(300000, 300000);
+        await Task.Delay(TimeSpan.FromSeconds(15));
```

Comment:
It may be helpful to put a comment that why this is being done. (delay as well as 15 seconds)

Reply:
Please add a TODO here and link to the P1 issue that you need to create. Something like:

`TODO: This is a workaround while isuee <issue_URL> is addressed`.

The issue should be a P1 type:process, should be assigned to the LEX team, and should be addresed as soon as possible.

Reply:
Please, a GitHub issue, any user stumbling with this comment should be able to know what the issue is about. The GitHub issue will be mirrored back to Buganizer and you can include that in normal planning, etc.